<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QWidget" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>373</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>373</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>替换</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>查找内容：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="content"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>替换为：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="replaceContent"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>全字匹配</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>区分大小写</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="findNext">
         <property name="text">
          <string>查找下一个</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replaceOnce">
         <property name="text">
          <string>替换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replaceAll">
         <property name="text">
          <string>全部替换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
